providers/aws/resources/discovery_conversion.go

```diff
@@ -710,6 +722,7 @@ func EbsConnectAsset(args []string, opts map[string]string) *inventory.Asset {
 	asset.IdDetector = []string{ids.IdDetector_Hostname} // do not use cloud detect or host key here
 	asset.Connections = []*inventory.Config{{
 		Type:     string(awsec2ebsconn.EBSConnectionType),
+		Backend:  inventory.ProviderType_AWS_EC2_EBS,
```

Comment:
nit: Isn't this deprecated?

Reply:
oh ya i can remove that, i think i placed it there while testing